docs/dev-tools/chopsticks/overview.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Using Chopsticks for Substrate-based Chains
+description: Configure and fork Substrate-based blockchains locally with Chopsticks. Learn installation, configuration, and usage for streamlined blockchain development.
+---
+
+# Chopsticks
+
+## Introduction
+
+[Chopsticks](https://github.com/AcalaNetwork/chopsticks/){target=_blank}, created and maintained by the [Acala Foundation](https://github.com/AcalaNetwork){target=_blank}, is a powerful tool designed to enhance the development process for Substrate-based blockchains. It offers developers a user-friendly method to locally fork existing chains, enabling them to:
+
+- Experiment with custom blockchain configurations in a local environment
+- Replay blocks and analyze how extrinsics affect state
+- Fork multiple blocks for comprehensive XCM testing
+
+With Chopsticks, developers can simulate and test complex blockchain scenarios without deploying to a live network. This tool significantly reduces the complexity of building blockchain applications on Substrate, making it more accessible to developers of varying experience levels. Ultimately, Chopsticks aims to accelerate innovation in the Substrate ecosystem by providing a robust, flexible testing framework.
+
+!!!note
+    Chopsticks currently does not support Ethereum JSON-RPC calls, so you cannot use it to fork your chain and connect Metamask.
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+- [Node.js](https://nodejs.org/en/){target=_blank}
+- Package manager - [npm](https://www.npmjs.com/){target=_blank} should be installed with Node.js by default. Alternatively, you can use other package managers like [Yarn](https://yarnpkg.com/){target=_blank}
+
+## Getting Started
+
+You can install Chopsticks either globally or locally in your project. Choose the option that best fits your development workflow.
+
+### Global Installation
+
+To install Chopsticks globally, allowing you to use it across multiple projects, run:
+
+```bash
+npm i -g @acala-network/chopsticks
+```
+
+Now, you should be able to run the `chopsticks` command from your terminal.
+
+### Local Installation
+
+To use Chopsticks in a specific project, first create a new directory and initialize a Node.js project:
+
+```bash
+mkdir my-chopsticks-project
+cd my-chopsticks-project
+npm init -y
+```
+
+Then, install Chopsticks as a local dependency:
+
+```bash
+npm i @acala-network/chopsticks
+```
+
+Finally, you can run Chopsticks using the `npx` command:
+
+```bash
+npx @acala-network/chopsticks
```

Comment:
Is NPX a standard thing? Or it is something you need to install globally? If so, we need to add that here or just run the binary the non NPX way

Reply:
NPX comes automatically installed with NPM. In this example, they create a project from scratch and do not need to install anything else.